### ISSUE TITLE
Color for Custom title and menu bar

### ISSUE THREAD

**Issue by ritwickdey** [Binaryify/OneDark-Pro#265]
There are new theme colors for Custom title and menu bar

```html
menu.separatorBackground: Color of a separator menu item in menus.
sideBarSectionHeader.border: Side bar section header border color.
inputValidation.infoForeground: Input validation foreground color for information severity.
inputValidation.warningForeground: Input validation foreground color for warning severity.
inputValidation.errorForeground: Input validation foreground color for error severity.
editor.snippetTabstopHighlightBackground: Highlight background color of a snippet tabstop.
editor.snippetTabstopHighlightBorder: Highlight border color of a snippet tabstop.
editor.snippetFinalTabstopHighlightBackground: Highlight background color of the final tabstop of a snippet.
editor.snippetFinalTabstopHighlightBorder: Highlight border color of the final tabstop of a snippet.
activityBar.inactiveForeground: Activity bar item foreground color when it is inactive.
```

* Link1: https://code.visualstudio.com/updates/v1_28#_custom-title-and-menu-bar-on-windows-by-default

* Link2: https://code.visualstudio.com/updates/v1_28#_new-theme-colors

**Comment by Binaryify** [comment]
I just fix it in v2.17.4, you can update and check if it right now

**Comment by ritwickdey** [comment]
yup. Thanks. It looks nice 💘
